bundles/io/org.openhab.io.gcal/src/main/java/org/openhab/io/gcal/internal/util/ExecuteCommandJob.java

```diff
@@ -44,18 +52,65 @@
     public void execute(JobExecutionContext context) throws JobExecutionException {
         String content = (String) context.getJobDetail().getJobDataMap().get(JOB_DATA_CONTENT_KEY);
 
+        ItemRegistry registry = GCalActivator.itemRegistryTracker.getService();
+        EventPublisher publisher = GCalActivator.eventPublisherTracker.getService();
+
+        if (registry == null) {
+            logger.warn("Sorry, no item registry service available!");
+            return;
+        }
+
+        if (publisher == null) {
+            logger.warn("Sorry, no event publisher service available!");
+            return;
+        }
+
+        if (content.startsWith("[PresenceSimulation]")) {
+            try {
+                Item item = registry.getItem("PresenceSimulation");
+                if (item.getState() != OnOffType.ON) {
+                    logger.info(
+                            "Presence Simulation job detected, but PresenceSimulation is not in ON state. Job is not executed");
```

Comment:
I think this could be reduced to debug, otherwise there might be many unneeded log entries.  WDYT?